data/css/meson.build

```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Endless Mobile, Inc.
+#
+# We only need to have this file in this subdirectory
+# as outputs to custom_target cannot have path separators
+# their names.
+
+sass = find_program('sassc')
```

Comment:
It looks like this isn't actually used - you could remove it, or it might be good to pass `sass.path()` as an extra argument to `meson_sassc` if you want to support the `sassc` binary being in a nonstandard location.